### PR TITLE
added Github issue template

### PR DESCRIPTION
**Pull request by metadave**
See also: https://github.com/blog/2111-issue-and-pull-request-templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+In order to help us troubleshoot the issue, please be sure to include the following information if applicable:
+
+- [ ] The OS (including version) where you are running any of the Habitat commands
+- [ ] Debug/backtrace of the command you are trying to run. You can set the following environment variables before running the `hab` command to generate a trace:
+
+		RUST_LOG=debug RUST_BACKTRACE=1 hab ...
+- [ ] Current Habitat environment variables. These can be gathered using:
+
+		env | grep HAB
```
Comment (smith) [inline .github/ISSUE_TEMPLATE.md]: people should not include their `HAB_AUTH_TOKEN`.

Comment (metadave) [inline .github/ISSUE_TEMPLATE.md]: whoa, good catch!

Comment (metadave) [inline .github/ISSUE_TEMPLATE.md]: fixed and included a note in bold/italics not to include that env var. Do you think there is a better way to collect env vars?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Hello - thanks for reporting an issue with Habitat.
+
+In order to help us troubleshoot the issue, please be sure to include the following information if applicable:
+
+- [ ] The OS (including version) where you are running any of the Habitat commands
+- [ ] Debug/backtrace of the command you are trying to run. You can set the following environment variables before running the `hab` command to generate a trace:
+
+		RUST_LOG=debug RUST_BACKTRACE=1 hab ...
+- [ ] Current Habitat environment variables. These can be gathered using:
```
Comment (miketheman) [inline .github/ISSUE_TEMPLATE.md]: Clarify if this is inside the studio or on the host platform?

Comment (metadave) [inline .github/ISSUE_TEMPLATE.md]: good call, thanks

**Comment by miketheman** [comment]
gif-keyboard-17204468114899162803

**Comment by metadave** [comment]
lets try this out and see if it helps

thanks for the review @miketheman 

@thesentinels r+

**Comment by thesentinels** [comment]
:pushpin: Commit 888ee24 has been approved by `metadave`

**Comment by thesentinels** [comment]
:hourglass: Testing commit 888ee24 with merge 8d04902...

**Comment by thesentinels** [comment]
:sunny: Test successful - [travis](https://travis-ci.org/habitat-sh/habitat/builds/144227984)

**Pull request merged**